### ISSUE TITLE
Updated files are giving an Import errors?

### ISSUE THREAD

**Issue by puternutt** [sealemar/BingRewards#299]
I used git.exe to clone a git folder using this command:

"C:\Program Files\Git\cmd\git.exe" clone https://github.com/sealemar/BingRewards.git "C:\BingRewards-master\git"

in doing so I received the following script errors:

                   0                100
Progress counter : Traceback (most recent call last):
  File "C:\BingRewards-master\main.py", line 24, in <module>
    from bingRewards import BingRewards
  File "C:\BingRewards-master\pkg\bingRewards.py", line 18, in <module>
    import bingDashboardParser as bdp
  File "C:\BingRewards-master\pkg\bingDashboardParser.py", line 19, in <module>
    from bs4 import BeautifulSoup
ImportError: No module named bs4

What's going on here anyone? was is this module it's asking for bs4?

**Comment by elisiano** [comment]
The dependency on bs4 is a fairly recent change. 
You can install all the dependencies (which for now is only bs4) with
‘’’
pip install -r requirements.txt
‘’’
(to be executed from the repo path)

**Comment by specu** [comment]
if you unzip this file https://www.crummy.com/software/BeautifulSoup/bs4/download/4.6/beautifulsoup4-4.6.0.tar.gz and copy the bs4 folder from within it so that main.py and folder bs4 live in the same folder that should do it. you dont have to go through installation. tested only on linux.

if the licenses are compatible it would be nice to just incorporate bs4 into bingrewards source.

**Comment by puternutt** [comment]
Thanks @specu and @elisiano for the replies. Specu, I did download it as you indicated and placed the bs4 in the root of the master folder, error went away. Now I get this: 

ConfigError: Corresponding account is not found for EVENTS.ACCOUNT.ref = [Live_gogo@hotmail.com]

which I believe is the default account given in the config file. I've removed that account and replaced that account with my account. So not sure why it's doing that.
